moveit_planners/ompl/ompl_interface/src/ompl_interface.cpp

```diff
@@ -183,7 +183,8 @@ void ompl_interface::OMPLInterface::loadPlannerConfigurations()
   for (std::size_t i = 0; i < group_names.size(); ++i)
   {
     // the set of planning parameters that can be specific for the group (inherited by configurations of that group)
-    static const std::string KNOWN_GROUP_PARAMS[] = { "projection_evaluator", "longest_valid_segment_fraction" };
+    static const std::string KNOWN_GROUP_PARAMS[] = { "projection_evaluator", "longest_valid_segment_fraction",
+                                                      "force_joint_model_state_space" };
```

Comment:
i don't like the term "force" because i keep thinking you are referring to power/torque. how about "mandate_joint_model_state_space"?

Reply:
i cannot find the file,
`moveit_planners/ompl/ompl_interface/src/ompl_interface.cpp` in my ROS. Have i missed to install some package?